requirements/constraints.txt

```diff
@@ -7,12 +7,3 @@
 # link to other information that will help people in the future to remove the
 # pin when possible.  Writing an issue against the offending project and
 # linking to it here is good.
-
-
-# This file contains all common constraints for edx-repos
```

Comment:
I will implement proper way of syncing common constraint to local.
This is temporary.